superflore/generators/bitbake/run.py

```diff
@@ -188,7 +189,7 @@ def main():
         delta = gen_delta_msg(total_changes)
         missing_deps = gen_missing_deps_msg(total_broken)
         # Commit changes and file pull request
-        overlay.commit_changes(args.ros_distro)
+        overlay.commit_changes('all' if args.all else args.ros_distro)
```

Comment:
I think this is the only change in this PR that is fixing an issue? If you could please file an issue for that problem as well that would be really great (for tracking/changelog generation for the next release).

Reply:
Yes, thanks! Fixes #183